.github/workflows/publish.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish Bintray
+
+on:
+  create:
+    tags:
+      - v*
+    
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+   
+    - name: Create Artifacts
+      run: ./gradlew install
```

Comment:
clean and build are not needed?

Reply:
no, its not, nobody do that in their actions

Reply:
ok, for some reason we have it in our guide how to publish jcenter is to clean `clean` `assemble`  `install` `bintrayUpload` but maybe install has `assemble` as prerequisity and `clean` is just nice to have